src/store.js

```diff
@@ -100,6 +100,12 @@ export class PortalStore {
     return this.api.fetchData('characters');
   }
 
+/*
```

Comment:
Some leftovers?

Reply:
Not leftovers, something to be implemented. I can remove it, but intend to add it in the future anyway.

Reply:
It should at least be commented as TODO if this is really planned, but apart from that, it's better not to push such stuff to the main branch